plugins/coverbrowser.koplugin/bookinfomanager.lua

```diff
@@ -544,6 +544,7 @@ function BookInfoManager:collectSubprocesses()
             local pid = self.subprocesses_pids[i]
             if util.isSubProcessDone(pid) then
                 table.remove(self.subprocesses_pids, i)
+                UIManager:allowStandby()
```

Comment:
We may have multiple pids to collect (`while i <= #self.subprocesses_pids` - I dont remember if this really happens, but the code allows for it), so allowing standby after only one collected might be wrong. Best to do it between original lines 575/576.

Reply:
This is the opposite part of the "1 pid = 1 ban" logic. The allow is called for each pid that terminated either on its own, or got explicitly nuked by terminateBackgroundJobs() at some essentially *random* point earlier.